tests/ui/suggestions/suggest-type-for-const.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0121]: the placeholder `_` is not allowed within types on item signatures for constants
+  --> $DIR/suggest-type-for-const.rs:7:10
+   |
+LL | const C: _ = Wrapper(foo);
+   |          ^
+   |          |
+   |          not allowed in type signatures
+   |          help: replace with the correct type: `Wrapper<fn()>`
```

Comment:
This test doesn't actually test new behaviour, as the output on stable, beta and nightly is already the same:

```
error[E0121]: the placeholder `_` is not allowed within types on item signatures for constants
 --> src/main.rs:5:10
  |
5 | const C: _ = Wrapper(foo);
  |          ^
  |          |
  |          not allowed in type signatures
  |          help: replace with the correct type: `Wrapper<fn()>`
```